.github/workflows/conan.yaml

```diff
@@ -0,0 +1,19 @@
+name: build conan package
+on: push
+jobs:
+  build_conan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # install dependencies
+      - name: dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -yq python3 python3-pip python3-setuptools build-essential cmake make
+          sudo pip3 install conan
```

Comment:
You can use [`actions/setup-python`](https://github.com/actions/setup-python) + `pip` to install python and conan.
With that change, you can test libconfig on all os'es (`ubuntu-latest` + `windows-latest` + `macos-latest`)